src/api/status/package.json

```diff
@@ -18,10 +20,17 @@
     "express": "^4.17.1"
   },
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=12.0.0 <=14"
   },
   "devDependencies": {
     "env-cmd": "^10.1.0",
-    "nodemon": "^2.0.15"
+    "nodemon": "^2.0.15",
+    "gulp": "^4.0.2",
+    "gulp-autoprefixer": "^7.0.1",
+    "gulp-clean": "^0.4.0",
+    "gulp-install": "^1.1.0",
+    "gulp-sass": "^4.1.0",
+    "gulp-sourcemaps": "^2.6.5",
+    "gulp-open": "^3.0.1"
```

Comment:
These can't be `devDependencies` since we [don't install them in production](https://github.com/Seneca-CDOT/telescope/blob/17d5c1522e7b799d4d9b826ec0223c4bb3714ea9/src/api/status/Dockerfile#L10) (this is probably why the `e2e` tests are failing).

Reply:
That makes sense, I'll move them.